http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ZombieMonitorQueries.scala

```diff
@@ -3,41 +3,53 @@ package db
 
 import cats.data.Chain
 import cats.implicits._
+import org.broadinstitute.dsde.workbench.leonardo.config.Config
 import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.api._
 import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.mappedColumnImplicits._
+import org.broadinstitute.dsde.workbench.leonardo.db.clusterQuery.unmarshalCluster
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 import slick.dbio.DBIO
 
 import scala.concurrent.ExecutionContext
 
 object ZombieMonitorQueries {
 
-  def listZombieQuery(implicit ec: ExecutionContext): DBIO[Map[GoogleProject, Chain[PotentialZombieRuntime]]] = {
-    val clusters = clusterQuery.filter(_.status inSetBind RuntimeStatus.activeStatuses.map(_.toString))
-    val joinedQuery = clusters.joinLeft(RuntimeConfigQueries.runtimeConfigs).on(_.runtimeConfigId === _.id)
-    joinedQuery.result.map { cs =>
-      cs.toList.foldMap { c =>
+  def listActiveZombieQuery(implicit ec: ExecutionContext): DBIO[Map[GoogleProject, Chain[ZombieCandidate]]] = {
+    val runtimes = clusterQuery.filter(_.status inSetBind RuntimeStatus.activeStatuses.map(_.toString))
+    val joinedQuery = runtimes.joinLeft(RuntimeConfigQueries.runtimeConfigs).on(_.runtimeConfigId === _.id)
+    joinedQuery.result.map { rs =>
+      rs.toList.foldMap { r =>
+        val clusterRec = r._1
+        // .get here should be okay since every cluster record in database should have a runtimeConfig
+        val cloudService = r._2.get.runtimeConfig.cloudService
+        val runtime = unmarshalCluster(clusterRec, Seq.empty, List.empty, Map.empty, List.empty, List.empty, List.empty, List.empty)
         Map(
-          c._1.googleProject -> Chain(
-            PotentialZombieRuntime(
-              c._1.id,
-              c._1.googleProject,
-              c._1.clusterName,
-              RuntimeStatus.withName(c._1.status),
-              c._1.auditInfo,
-              c._2.get.runtimeConfig.cloudService
-            ) // .get here should be okay since every cluster record in database should have a runtimeConfig
+          r._1.googleProject -> Chain(
+            ZombieCandidate(runtime, cloudService)
           )
         )
       }
     }
   }
 
+  private def unconfirmedRuntimeLabels: Query[LabelTable, LabelRecord, Seq] =
+    labelQuery.filter(_.key === Config.zombieClusterMonitorConfig.deletionConfirmationLabelKey).filter(_.value === "false")
```

Comment:
Existing records won't have this label key at all.. does this code take that into account?